copilot/copilot/app.py

```
@@ -0,0 +1,34 @@
+            template=client.V1PodTemplateSpec(
+                metadata={"labels": {"app": name}},
+                spec=client.V1PodSpec(
+                    containers=[client.V1Container(name=name, image=image)]
+                )
+            ),
+        )
+    )
+
+    try:
+        v1.create_namespaced_deployment(namespace="default", body=deployment)
+        return {"message": f"Deployment {name} created successfully"}
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)